src/middlewared/middlewared/plugins/cloud_sync.py

```diff
@@ -638,7 +638,11 @@ class Config:
         Bool("filename_encryption", default=False),
         Str("encryption_password", default=""),
         Str("encryption_salt", default=""),
-        Cron("schedule", required=True),
+        Cron(
+            "schedule",
+            Str("minute", default="00"),
```

Comment:
Wouldn'it it be better to pass defaults as dictionary instead of this. What will happen if I say `Int("minute", default=0),`? Or `Str("minuete", default="00"),`?

Reply:
@themylogin This will certainly cover mistakes made on our part, but it would differ from the schema standard we have laid out. Do you think i should proceed with this ? If yes, I'll move ahead and take care of this :)
If not the former case can be taken care of quite easily by only taking defaults into consideration but then again it depends on how we choose to implement it. Please let me know what you think. Thank you

Reply:
I don't think we have any schema standard we have laid out related to this. Your proposed `Cron` definition looks like `Dict` definition, that's fancy, but IMO strange. @william-gr what do you think?

Reply:
I agree with @themylogin here, redefining whole Schema type within Cron seems strange, simply overriding defaults seems saner.

Reply:
Perfect. Thank you @themylogin @william-gr. I'll have this covered asap